homeassistant/components/environment_canada/camera.py

```diff
@@ -78,17 +78,12 @@ def name(self):
         """Return the name of the camera."""
         if self.camera_name is not None:
             return self.camera_name
-        return " ".join([self.radar_object.station_name, "Radar"])
+        return "Environment Canada Radar"
```

Comment:
The name change also looks like a breaking change. It will affect entity_ids if there's no unique id set for the entities.